CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Increase Medialive profiles segment to 4 seconds
 - Increase Mediapackage endpoint segment to 4 seconds
 - Rollback Medialive control rate mode to CBR
+- Static files are managed using whitenoise
```

Comment:
Should you also remove creating the S3 bucket in Terraform?

Reply:
We should but this will also delete the bucket. In a blue/green strategy this is a BC break. 
Should we jump to version 4 ? Wait few releases before removing it ?

Reply:
Worst, in a blue/green stategy, the current branch will not work once the next one deployed. IMO we should keep it for now and remove it in the next major release.

Reply:
Yes agreed! Open an issue to remember maybe.

Reply:
https://github.com/openfun/marsha/issues/883